src/components/UserProfile/UserProfileAccount/MetamaskDetails/index.js

```diff
@@ -18,7 +18,7 @@ import { Networks } from "../../../../config/Networks";
 
 class MetamaskDetails extends Component {
   state = {
```

Comment:
Is this needed?

Reply:
Yeah. I'm setting withdraw tab as the default tab instead of the deposit tab.